libimagequant.c

```diff
@@ -543,6 +546,8 @@ static liq_image *liq_image_create_internal(const liq_attr *attr, liq_color* row
         if (!img->temp_row) return NULL;
     }
 
+    img->temp_f_row = NULL;
```

Comment:
I don't understand why is it nulled here. If it was null already (as should have been from the init earlier) then it's unnecessary. If it wasn't null, that's a leak.